content/docs/gto/user-guide/dvc.md

```diff
@@ -0,0 +1,144 @@
+# GTO with DVC
+
+In many ML projects, data isn't stored in a Git repository and needs to be
+downloaded from external sources. [DVC](https://dvc.org) is a common way to
+store binaries for the artifacts registered with GTO.
+
+<details>
+
+### Learn about different approaches to this
+
+1. You can commit artifacts to Git repo. If they aren't small enough, this is
+   not recommended. To bypass this limitation, you can use
+   [Git-lfs](https://git-lfs.github.com).
+2. You can version binaries with [DVC](https://dvc.org/) and commit pointers to
+   them to the repo. This is the recommended approach for large files.
+3. You can version binaries manually somewhere, specifying URL to them as `path`
+   in `artifacts.yaml`. This can be done, if your artifacts are already
+   versioned by some external system.
+
+</details>
+
+If you are new to DVC, you should start with
+[DVC Get Started](https://dvc.org/doc/start) first, and then come back to this
+Guide.
+
+<!-- ```
+
+dvc init --no-scm dvc remote add az azure://container-name export
+AZURE_STORAGE_CONNECTION_STRING='YOUR_CONNECTION_STRING'
+
+dvc import-url --no-download azure://container-name/data.parquet
+
+``` -->
+
+## Tracking an artifact with DVC
+
+First, we need to start tracking artifact with DVC. If you produce this artifact
+in DVC Pipelines, it's done automatically.
+
+If the artifact is located inside your Git repo, you can use `dvc add`:
+
+```cli
+$ dvc add model.pkl
+$ git add model.pkl.dvc
+```
+
+If the artifact is located in some external storage, we can use `dvc import-url`
+to still keep metainformation about it in the repo (use `--no-download` to skip
+downloading it):
+
+```cli
+$ dvc import-url --no-download s3://container/model.pkl
+$ git add model.pkl.dvc
+```
+
+## Annotating DVC-tracked artifacts
+
+Once the artifact is tracked with DVC within your repo, we can annotate it with
+GTO:
+
+```cli
+$ gto annotate model --path model.pkl
+```
+
+This will modify `artifacts.yaml`, adding:
+
+```yaml
+model:
+  path: model.pkl
+```
+
+Now you should commit changes to Git, and you can register versions and assign
+stages referencing the new commit.
+
+```cli
+$ git add artifacts.yaml
+$ git commit -m "version artifact binaries with DVC and annotate it with GTO"
```

Comment:
````suggestion
Commit the changes to Git in order to `gto register` artifact versions and
`gto assign` them to deployment stages referencing the new commit.

```cli
$ git add artifacts.yaml
$ git commit -m "version artifact binaries with DVC and annotate it with GTO"
````

Reply:
Why do you want to reference a commit in `gto register/assign` though? I'm not sure this is clear (maybe link to the UG index page section where this is explained? Or elaborate here perhaps)